### PR TITLE
Cleanup obsoleted jobs

### PR DESCRIPTION
**Pull request by liangxia**
`openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml` is the old way to do multi-jump upgrade, it has been replaced ( and covered ) by the chained upgrade.

/cc @jiajliu @JianLi-RH @shellyyang1989 @jianlinliu

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml [hunk 1]
```diff
@@ -1,93 +0,0 @@
...
-      - env: RELEASE_IMAGE_INTERMEDIATE
-        name: release:intermediate
-      - env: RELEASE_IMAGE_TARGET
-        name: release:target
-      from: cli
-      resources:
-        requests:
-          cpu: 100m
-          memory: 200Mi
-    - chain: cucushift-upgrade
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml:60]: Shall we also remove the chain file together?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml [hunk 2]
```diff
@@ -1,93 +0,0 @@
...
-      - env: RELEASE_IMAGE_INTERMEDIATE
-        name: release:intermediate
-      - env: RELEASE_IMAGE_TARGET
-        name: release:target
-      from: cli
-      resources:
-        requests:
-          cpu: 100m
-          memory: 200Mi
-    - chain: cucushift-upgrade-disconnected
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml:87]: Shall we also remove the chain file together?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml (hunk 1 above)
Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml:60]: I think that can be done in a separate PR with extra check.

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml:60]: I did some local check, the `cucushift-upgrade` chain is only referenced by this ci yaml file.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml:60]: Done

**Comment by shellyyang1989** [comment]
/lgtm 

sorry for the late.

**Pull request merged**